### PR TITLE
Fixes for CHiME-7 DASR recipe ASR training

### PR DESCRIPTION
**Pull request by popcornell**
I have fixed some issues (bash script errors) encountered when training the ASR baseline model from scratch. 
I have also added for the gss a check whether the GPUs are in exclusive thread mode.

### CODE REVIEW

File: egs2/chime7_task1/asr1/run.sh [hunk 1]
```diff
@@ -120,16 +128,20 @@ if [ ${stage} -le 1 ] && [ $stop_stage -ge 1 ]; then
...
+      if [ "$decode_train" == "train" ] && [ $dset_part == "eval" ]; then
         continue
       fi
 
+      #if [ ! -d $chime7_root/$dset/audio/$dset_part ]; then
+      #  log "Skipping $dset $dset_part because it does not exist on disk. This is
+      #  fine if you don't have evaluation set yet."
+      #  continue
+      #fi
+
```
Comment (sw005320) [inline egs2/chime7_task1/asr1/run.sh:144]: Do you want to keep these lines?

Comment (popcornell) [reply egs2/chime7_task1/asr1/run.sh:144]: We can remove it. 
I have added a new flag for skipping the evaluation. 
Let me update the PR

**Comment by popcornell** [comment]
Yes you can merge now

**Comment by sw005320** [comment]
Thanks!

**Pull request merged**